### ISSUE TITLE
proposed changes to Italy

### ISSUE THREAD

**Issue by KabelWlan** [tmrowco/electricitymap-contrib#3205]
Hi everyone

During my investigations on updating the capacities in Italy (PR follows later) I found that there were some changes to the zones.

Zones 2020:
![Italy 2020](https://user-images.githubusercontent.com/49598085/124389925-ad6ca000-dce9-11eb-8d70-85e114f7b0c2.png)

Zones 2021:
![Italy 2021](https://user-images.githubusercontent.com/49598085/124389929-b198bd80-dce9-11eb-8d16-de47f75ac3fa.png)

The first change is that the region Umbria is no longer part of IT-CNO. 
It is now part of IT-CSO. I contacted Entsoe and they told me that the published data for IT-CSO now includes the region Umbria and IT-CNO doesn't anymore.
This is what Entsoe told me:

![E-Mail Entsoe](https://user-images.githubusercontent.com/49598085/124390036-1c49f900-dcea-11eb-8fc4-2bd140086f41.png)

The second point is that we added the region "Calabria" to "Italy South" in #3053. I checked Entsoe and there is everything available to separate the zone IT-South into IT-South and IT-Calabria. If the data is available why not split it down further to get a higher resolution.

What do you guys think?

**Comment by alixunderplatz** [comment]
Hi @KabelWlan,
seems like this hasn't been implemented yet to the code.
Both points regarding Umbria moving from IT-CNO to IT-CSO and splitting Calabria from IT-SO and all the related exchange arrows and zone geometries should be fixed asap.

**Comment by pierresegonne** [comment]
Don't they make these updates every couple of years?

I think that updating the parsers and geographies is OK, as our historical data probably contains some of these switches already, we just need to add a disclaimer somewhere to explain that phenomenon

**Comment by Kongkille** [comment]
Yes the updates seem to occur occasionally, I am specifically thinking of splitting out IT-SO into two separate zones one containing "South" and the other containing "Calabria". Doing this, we would essentially lose our current historical data for IT-SO as a whole.

**Comment by VIKTORVAV99** [comment]
Now that we have an aggregated view in the works does it make sense to split the South zone into South and Calabria like shown on this map?
We obviously wouldn't have historical data for the new zone, but it would increase the data resolution for the zone view and for the country view the data should be the same.

**Comment by pierresegonne** [comment]
Well the aggregation does not really work for arbitrary granularities of zones. We right now try to stick with the definition of zone / country. 

This means that if we split up South in South and Calabria, we won't be able to re-aggregate into the former south and offer continuity of data. 

I'm a bit concerned about these constantly shifting definitions of zones in Italy, as it will make it impossible for us to offer reliable granular historical data.

**Comment by VIKTORVAV99** [comment]
> Well the aggregation does not really work for arbitrary granularities of zones. We right now try to stick with the definition of zone / country.
> 
> This means that if we split up South in South and Calabria, we won't be able to re-aggregate into the former south and offer continuity of data.
> 
> I'm a bit concerned about these constantly shifting definitions of zones in Italy, as it will make it impossible for us to offer reliable granular historical data.

This will mostly likely be an issue for Sweden as well in the coming years as there are talks about restructuring the existing bidding zones or combining some of them. We should probably try and figure out a solution for it. I have a few ideas we can talk a bit more about later if you wish.

**Comment by Gianfilippo980** [comment]
If the only change was the division of Calabria from South, it would be conceivable to update the map and show the same data for both zones when displaying older data. But as Umbria changed zone it would be hard to display Centre-North and Centre-South.
But, on the other hand, as time goes by, I see the problem of displaying the current data as increasingly urgent in comparison to the older archive.

**Comment by VIKTORVAV99** [comment]
@FelixDQ what do you think about this?

**Comment by Gianfilippo980** [comment]
Also, I wonder, how are the data currently shown being converted?

**Comment by edo2313** [comment]
Do you have any updates on this? As the zones are different between ElectricityMaps and Terna/GME, if you want to calculate the electricity price and also the co2 produced you (sometimes) have to use 2 different zones.

**Comment by Gianfilippo980** [comment]
The current policy seems to be that Calabria is aggregated to the South, as the project couldn't display historic data with revised zones.

https://github.com/electricitymaps/electricitymaps-contrib/issues/7891#issuecomment-2866220951
